src/test/java/com/google/api/codegen/testdata/discoveries/php/php_storage.v1.json.baseline

```diff
@@ -1172,51 +1034,44 @@ echo '<pre>', var_export($response, true), '</pre>', "\n";
  *    https://console.developers.google.com/apis/api/storage
  * 2. This sample uses Application Default Credentials for authentication.
  *    If not already done, install the gcloud CLI from
- *    https://cloud.google.com/sdk/ and run
+ *    https://cloud.google.com/sdk and run
  *    `gcloud beta auth application-default login`
+ *    For more information, see:
+ *    https://developers.google.com/identity/protocols/application-default-credentials
  * 3. Install the PHP client library with Composer. Check installation
  *    instructions at https://github.com/google/google-api-php-client.
  */
 
 // Autoload Composer.
 require_once __DIR__ . '/vendor/autoload.php';
 
-// Use Application Default Credentials for authentication when running locally.
-// For more information, see:
-// https://developers.google.com/identity/protocols/application-default-credentials
 $client = new Google_Client();
 $client->setApplicationName('Google-StorageSample/0.1');
 $client->useApplicationDefaultCredentials();
 $client->addScope('https://www.googleapis.com/auth/cloud-platform');
 
 $service = new Google_Service_Storage($client);
 
-
-// TODO: Change placeholders below to desired parameter values for the `copy` method:
-
 // Name of the bucket in which to find the source object.
-$sourceBucket = '{MY-SOURCE-BUCKET}';
+$sourceBucket = '';  // TODO: Update placeholder value.
 
-// Name of the source object. For information about how to URL encode object names to be path safe, see
-// Encoding URI Path Parts.
-$sourceObject = '{MY-SOURCE-OBJECT}';
+// Name of the source object. For information about how to URL encode object names to be path safe,
+// see Encoding URI Path Parts.
+$sourceObject = '';  // TODO: Update placeholder value.
 
-// Name of the bucket in which to store the new object. Overrides the provided object metadata's bucket
-// value, if any.For information about how to URL encode object names to be path safe, see Encoding URI
-// Path Parts.
-$destinationBucket = '{MY-DESTINATION-BUCKET}';
+// Name of the bucket in which to store the new object. Overrides the provided object metadata's
+// bucket value, if any.For information about how to URL encode object names to be path safe, see
+// Encoding URI Path Parts.
+$destinationBucket = '';  // TODO: Update placeholder value.
 
 // Name of the new object. Required when the object metadata is not otherwise provided. Overrides the
 // object metadata's name value, if any.
-$destinationObject = '{MY-DESTINATION-OBJECT}';
-
+$destinationObject = '';  // TODO: Update placeholder value.
 
+// TODO: Assign values to desired properties of `requestBody`:
 $requestBody = new Google_Service_Storage_StorageObject();
-// TODO: Assign values to desired properties of the `requestBody` object.
-
-
-$optParams = array();
 
+$optParams = [];
 // TODO: To download media content, use:
```

Comment:
I don't mind, it seemed reasonable to not use one there. I mostly chose to drop it because there's no newline in
```
$optParams = [];
do {
  ...
```
Do you want me to keep the newline before the media content case?